nym-wallet/src/components/Delegation/DelegateModal.tsx

```diff
@@ -253,6 +254,9 @@ export const DelegateModal: React.FC<{
         hidden
         divider
       />
+      <Typography fontSize="smaller" sx={{ color: 'text.primary' }}>
```

Comment:
Use another ModalListItem like this 
`<ModalListItem label="Est. fee for this transaction will be calculated in the next page" value=""/>`